app/src/main/res/values/strings.xml

```diff
@@ -240,6 +240,7 @@
     <!-- Names of the toolbar keys-->
     <string name="copy" tools:keep="@string/copy" translatable="false">@android:string/copy</string>
     <string name="cut" tools:keep="@string/cut">Cut</string>
+    <string name="paste" tools:keep="@string/paste">Paste</string>
```

Comment:
There is a `paste` string similar to `copy` above. Using default Android strings is good for consistency, and doesn't require translation.

Reply:
Nice!